src/Search.js

```diff
@@ -3,74 +3,77 @@ import pokemon from './pokemon.js'
 import Sort from './Sort.js';
 import Searchbar from './Searchbar.js';
 import PokeList from './pokeList.js';
+import request from 'superagent';
+
 // import Dropdown from './Dropdown.js';
 
 
 export default class Search extends Component {
     state = {
         pokemon: pokemon,
         sortBy: 'pokemon',
-        sortOrder: 'ascending',
+        sortOrder: 'asc',
         filter: '',
         searchQuery: '',
+        loading: false,
     }
 
 
-    handlePokeOptionChange = (e) => {
+    getPokemon = async () => {
+        console.log('the user clicked', this.state.searchQuery, this.state.sortBy, this.state.sortOrder);
+        const data = await request.get(`https://pokedex-alchemy.herokuapp.com/api/pokedex?sort=${this.state.sortBy}&direction=${this.state.sortOrder}&${this.state.sortBy}=${this.state.searchQuery}`)
```

Comment:
I noticed the attack and defense aren't quite working. I think the second this.state.sortBy needs to be hardcoded as pokemon :)